corehq/apps/domain/views.py

```diff
@@ -2071,6 +2080,57 @@ def post(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
 
+class CaseSearchConfigView(BaseAdminProjectSettingsView):
+    urlname = 'case_search_config'
+    page_title = ugettext_lazy('Case Search')
+    template_name = 'domain/admin/case_search.html'
+
+    @method_decorator(domain_admin_required)
+    @use_bootstrap3
+    def dispatch(self, request, *args, **kwargs):
+        return super(CaseSearchConfigView, self).dispatch(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+
+        def unpack_fuzzies(query_dict):
```

Comment:
yipes! I'm guessing my helper methods on CaseSearchConfig were not that helpful?